### DISCUSSION TITLE
How can I run specific feature files parallel in webdriverio

### DISCUSSION THREAD

**Discussion by bharatsinghoh** [webdriverio/webdriverio#10082] [Technical Discussion]
If we want to run some specific tests in parallel,, so is there any way to do it in webdriverio?

**Comment by christian-bromann** [comment]
WebdriverIO automatically runs all tests in parallel, you can find more information at https://webdriver.io/docs/organizingsuites

**Reply by xinjiahui** [reply]
i have practiced fllow Official doc，but it not work；
i configed wido.conf.ts like blow  `  specs: [
    ["./test/specs/test_base.ts",
      "./test/specs/test_e2e.improved.ts"],
    "./test/specs/test_e2e.ts",
  ],` but got result is like `[0-0] PASSED in chrome(138) - file:///test/specs/test_base.ts
2026-01-13T10:44:00.543Z INFO @wdio/cli:launcher: Run onWorkerEnd hook

 "spec" Reporter:
------------------------------------------------------------------
[chrome 138.0.7204.251 mac #0-0] Running: chrome (v138.0.7204.251) on mac
[chrome 138.0.7204.251 mac #0-0] Session ID: c651ce95d1a81d97907fb2afa1778013
[chrome 138.0.7204.251 mac #0-0]
[chrome 138.0.7204.251 mac #0-0] » test/specs/test_base.ts
[chrome 138.0.7204.251 mac #0-0] 🔧 基础测试套件
[chrome 138.0.7204.251 mac #0-0]    ✓ ✅ VSCode服务应该正确启动
[chrome 138.0.7204.251 mac #0-0]    ✓ 🧩 插件市场测试 - 正确打开插件详情简介页面
[chrome 138.0.7204.251 mac #0-0]    ✓ coderider登陆成功
[chrome 138.0.7204.251 mac #0-0]    ✓ 🔐 CodeRider登录应该成功
[chrome 138.0.7204.251 mac #0-0]    ✓ 🏠 登录后首页内容验证
[chrome 138.0.7204.251 mac #0-0]
[chrome 138.0.7204.251 mac #0-0] 5 passing (27.8s)


Spec Files:      1 passed, 1 total (100% completed) in 00:00:30  

2026-01-13T10:44:00.544Z INFO @wdio/local-runner: Shutting down spawned worker
2026-01-13T10:44:00.795Z INFO @wdio/local-runner: Waiting for 0 to shut down gracefully
2026-01-13T10:44:00.796Z INFO @wdio/local-runner: shutting down`
i start script is `wdio run ./wdio.conf.ts --maxInstances=3`

**Reply by christian-bromann** [reply]
Can you provide a minimal reproduction case as a GitHub project?

**Reply by baisongchen** [reply]
Thanks for your reply. I've already added you to the [repository](https://github.com/baisongchen/vscodeExtensionTest). I'm also checking if it's a configuration issue on my end, or possibly a version problem. 
Also, this comment was posted by my friend because I couldn't comment yesterday due to network issues.

**Reply by baisongchen** [reply]
I found the problem; the AI assistant got the spec file name wrong.`./test/specs/test.e2e.improved.ts` thanks for help
